src/effects/native/reverbeffect.cpp

```diff
@@ -27,36 +27,61 @@ EffectManifest ReverbEffect::getManifest() {
     time->setId("bandwidth");
     time->setName(QObject::tr("Bandwidth"));
     time->setShortName(QObject::tr("BW"));
-    time->setDescription(QObject::tr("Higher bandwidth values cause more "
-            "bright (high-frequency) tones to be included"));
+    time->setDescription(QObject::tr("Bandwidth of the low pass filter at the input. "
+            "Higher values result in less attenuation of high frequencies."));
     time->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
     time->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
     time->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
-    time->setDefaultLinkType(EffectManifestParameter::LINK_LINKED);
-    time->setDefaultLinkInversion(EffectManifestParameter::LinkInversion::INVERTED);
-    time->setMinimum(0.0005);
-    time->setDefault(0.5);
-    time->setMaximum(1.0);
+    time->setMinimum(MIN_VALUE);
+    time->setDefault(MAX_VALUE);
+    time->setMaximum(MAX_VALUE);
+
+    EffectManifestParameter* decay = manifest.addParameter();
+    decay->setId("decay");
+    decay->setName(QObject::tr("Decay"));
+    decay->setDescription(QObject::tr("Lower decay values cause reverberations to die out more quickly."));
+    decay->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
+    decay->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
+    decay->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
+    decay->setMinimum(MIN_VALUE);
+    decay->setDefault(DEFAULT_DAMPING);
+    decay->setMaximum(MAX_VALUE);
 
     EffectManifestParameter* damping = manifest.addParameter();
     damping->setId("damping");
     damping->setName(QObject::tr("Damping"));
     damping->setDescription(QObject::tr("Higher damping values cause "
-            "reverberations to die out more quickly."));
+            "high frequencies to decay more quickly than low frequencies."));
     damping->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
     damping->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
     damping->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
-    damping->setMinimum(0.005);
-    damping->setDefault(0.5);
-    damping->setMaximum(1.0);
+    damping->setMinimum(MIN_VALUE);
+    damping->setDefault(MIN_VALUE);
+    damping->setMaximum(MAX_VALUE);
+
+    EffectManifestParameter* blend = manifest.addParameter();
```

Comment:
> what do you use the blend parameter for that the chain D/W doesn't give you?

Control over the individual effect. For all other effects, there is a way to set the metaknob so the effect is not heard. Without adding this control at the effect level for reverb, there is no way to turn the reverb off by turning the metaknob. The only way to turn it off is by turning the dry/wet all the way dry for the whole chain or toggling the enable button for the effect, which abruptly stops the effect.